corehq/apps/app_manager/templates/app_manager/partials/releases_table.html

```diff
@@ -105,6 +105,14 @@
                   </div>
               {% endif %}
               <div class="build-is-released">
+                  <!--ko if: is_released() == 'error' -->
+
+                  <span class="label label-danger"
+                        data-bind="visible: is_released() == 'error'">{% trans "Could not update" %}</span>
```

Comment:
is this check of `is_released() == 'error'` needed if the ko if is used outside of it?

Reply:
oops, meant to remove that. thanks, fixed.